resources/backend/deployment.yaml

```diff
@@ -20,7 +20,7 @@ spec:
     spec:
       containers:
         - name: backend
-          image: europe-docker.pkg.dev/kyma-project/dev/busola-backend:PR-2556
+          image: europe-docker.pkg.dev/kyma-project/dev/busola-backend:PR-2596
```

Comment:
you don't need to bump the backend (your changes are only related to frontend), please revert that